lib/datasource/golang-version/index.ts

```diff
@@ -0,0 +1,106 @@
+import { ExternalHostError } from '../../types/errors/external-host-error';
+import { cache } from '../../util/cache/package/decorator';
+import { regEx } from '../../util/regex';
+import { isVersion, id as semverVersioningId } from '../../versioning/semver';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, Release, ReleaseResult } from '../types';
+
+const lineTerminationRegex = regEx(`\r?\n`);
+const releaseBeginningChar = '{';
+const releaseTerminationChar = '},';
+const releaseDateRegex = regEx(
+  `Date\\{(?<year>\\d+),\\s+(?<month>\\d+),\\s+(?<day>\\d+)\\}`
+);
+const releaseVersionRegex = regEx(
+  `Version\\{(?<versionMajor>\\d+),\\s+(?<versionMinor>\\d+),\\s+(?<patch>\\d+)\\}`
+);
+
+export class GolangVersionDatasource extends Datasource {
+  static readonly id = 'golang-version';
+
+  constructor() {
+    super(GolangVersionDatasource.id);
+  }
+
+  override readonly defaultRegistryUrls = [
+    'https://raw.githubusercontent.com/golang/website/',
```

Comment:
```suggestion
    'https://github.com/golang/go',
```

We should point to go source repo. I didn't found a repo which changelogs. 😕
It is they only publish them on the website.

Reply:
Did you get this mixed up with `sourceUrl`?

Reply:
🤦‍♂️